lib/core/network/repository/transaction_history_repository.dart

```diff
@@ -11,9 +11,11 @@ class TransactionHistoryRepository {
 
   TransactionHistoryRepository({required this.service});
 
-  Future<Result<List<TransactionModel>, HyphaError>> getTransactions(String userAccount) async {
+  Future<Result<List<TransactionModel>, HyphaError>> getTransactions(String userAccount, bool transferOnly) async {
     try {
-      final Response response = await service.getTransactions(userAccount);
+      final Response response = transferOnly
+          ? await service.getTransferTransactions(userAccount)
+          : await service.getAllTransactions(userAccount);
```

Comment:
Wondering if this "if" statement is the best way to express "only fetch transfer"